### ISSUE TITLE
general suggestions - quite a few

### ISSUE THREAD

**Issue by Nightshadow935** [UncreatedStaff/UncreatedWarfare#87]
so, instead of writing 5 different tabs for issues, i figured it would be easier, and better to write just the one. so, here we go

1: incentive to objectives
currently objectives have very little meaning to them to most people. the amount of games I have seen where people are losing the game, the final objective is under threat in Refinery, and where are all your team mates? in the city, setting up an FOB at Mosque or BC tower. this is either due to a lack of care, or a lack of knowledge. assuming the first, a fix to this could be higher rewards for capturing/being at objectives. a 1.5x multiplier for killing an enemy on an objective, for attack and defense. a match reward, so at the end of the game, if your team wins, you get a bonus of 10c and 250xp (random numbers, just my idea). other measures need to be put in place in regards to getting people there, however, this could be a start

2: team killing
team killing feels like a small issue. though one thing that could be improved on with it, is team killing friendly vehicles. lets say a jet and a tank are on the battlefield. the jet flies over and accidentally bombs the tank. currently, the only compensation that the tank driver would get, is, at most a "sorry" from the jet, and sometimes, not even that. a fix to this, would be a compensation. lets say the jet bombs the tank. instead of nothing happening, the person who bombed the tank has to pay 24 credits (double the cost of the tank) to the person who bought it. they would also incur a xp loss, equivalent to what you would typically gain from destroying a tank.  say you gain 100xp normally, you would lose 100xp instead. this would make it so friendly fire is rarer, and looked at carefully

3: classes
most times when playing, there is always someone who has the heavy AT class - one of the most important in the game. being able to take down heavy vehicles without a TOW/Kornet, and making all enemy vehicles not be careless all the time. the issue with it though, is that most times, there is only one allowed per match. as soon as a new match starts, everyone rushes towards it, hoping they get it. the issue, mid game, is, imagine the enemy team has an LAV, Bradly, and a tank. all terrorizing your team. you ask again and again in chat who is the Heavy AT class, but get no response. a solution to this, would be to be able to see who is what class. when you do /req on the sign, it would say that "only X amount of people are allowed this class at once. the other people/person is *blank*". not to be rude to newer players, but I, and likely many others would feel better if they knew that someone competent and experienced was using the kit, and not someone who is only using the gun because they don't realize that their job is to destroy vehicles

4: tickets
this I feel has been brought up before but maybe one more time couldn't hurt. tickets are useful to see. a simple command even, such as /tickets to show how many tickets each team has would be beneficial. letting each team know how close they are to losing. it could be argued that "watching the ticket count suddenly drop though could tell you what the enemy is bringing out" and while yes, that could be the case, a work around could be to only display ticket count in intervals of 10, 20, 25, 50, etc. which would make it near impossible to find out what the enemy is bringing out for its vehicles

5: rangefinders and binoculars
rangefinders and binoculars are two very, VERY vital assets in the field. for example, when playing an AT class, the scope you have has an approximate ranging on the scope. now from what I understand it isn't always accurate, however, it is still something that is useful. being able to see how far away an enemy tank is, how far away that jet is, or if you should worry about that sniper on top of hotel. being able to see ranges are very useful, and I believe more than just the sniper should have access to one. kits such as the AT kits. binoculars are different. they don't display range, obviously, though they are just as useful. kits such as breacher, combat engineer, and possibly medic, should have access to a pair. although most people don't use them, i know many people who see them as a crucial device that can change how you play. being able to see the enemy is more important than anything

with that, that's my list. I look forward to hearing back on these ideas. I didn't want to ramble on and on about each, so, any questions, not just from staff but from anyone are appreciated

**Comment by HamzaXSniper** [comment]
Binoculars is such an important piece of equipment for an anti tank, When i first purchased my custom kit i put in binoculars for why not but now i see its very helpful to spot enemy vehicles and has made me more of a help to the team. also I see the credit loss/gain method can be used to transfer credits between players.

**Comment by Nightshadow935** [comment]
> Binoculars is such an important piece of equipment for an anti tank, When i first purchased my custom kit i put in binoculars for why not but now i see its very helpful to spot enemy vehicles and has made me more of a help to the team. also I see the credit loss/gain method can be used to transfer credits between players.

true, in regards to the credit loss/gain, however you also gotta think, 1, how many people actually are nice on this server, and 2, it would be very, VERY VERY slow, and also would lead to them likely getting banned, as they would be constantly killing friendly vehicles, and, more importantly, friendly players. the vehicles that cost credits have a longer respawn time. its purpose is to make it so killing friendlies is looked at more carefully, so you don't just drop bombs wherever

the binoculars though, thank you, I appreciate that someone else sees it the way I do

**Comment by Nightshadow935** [comment]
just wanted to comment on this again. wanted to know if any staff have seen it or anything

**Comment by benvalkin** [comment]
1) I don't really think capturing points really needs to be that incentivized, but we could increase the XP/credit reward for capturing and defending enemy points. Also, to make flag gamemodes more competitive and interesting, I've thought about implementing "double neutral" logic where you can still neutralize and enemy flag even if they neturalize your previous point (like in Squad).
2) I think a better solution to teamkilling in general would be reverse friendly fire like their is with anti-main camping.
3) Maybe with the new action menu we can add a "Need AT Support" button which notifies all nearby AT classes?
4) Enemy tickets are supposed to be hidden
5) We'll give all kits without scopes binoculars next season, we just haven't got around to doing it yet

Also, in future could you make separate issues (it's okay if you make 5 different ones), because we can't correctly tag, manage and close an issue with 5 different suggestions in it

**Comment by Nightshadow935** [comment]
alright. thank you for the reply and comments. the binoculars I feel are going to be really nice, along with the reverse friendly fire. however I can see that working in a negative sometimes, as I have experienced people who have asked "hey can you kill me real quick so I can go back to spawn". 

as for the AT issue, the issue I often face is having the new guy have the AT class, not knowing how to properly use it and instead wasting the rockets, or not even using them and running around with the MP5/UMP (which I think the UMP still needs a buff, since in comparison to the MP5 its complete shit). perhaps something that could help with this is implementing something similar to the vehicles, where certain kits are only for certain ranks. heavy AT, third level squad lead, medic, rifleman, only being able to be bought/requested at the second level (I forget what the ranking order is). it would hopefully make it so new players don't instantly rush towards the thing that just says "OOOO ROCKET LAUNCHER"
